From 1c789f5ffde5d443fa2ce6ccfc4eb55f9a8afb4c Mon Sep 17 00:00:00 2001
From: Florence Blanc-Renaud <flo@redhat.com>
Date: Tue, 25 Feb 2025 10:24:56 +0100
Subject: [PATCH] ipatests: use "sos report" instead of "sosreport" command

The "soscommand" has been deprecated and "sos report" should be
used instead. The redirector was removed in sos 4.9.

Fixes: https://pagure.io/freeipa/issue/9752
Signed-off-by: Florence Blanc-Renaud <flo@redhat.com>
Reviewed-By: Alexander Bokovoy <abokovoy@redhat.com>
---
 ipatests/test_integration/test_ipahealthcheck.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ipatests/test_integration/test_ipahealthcheck.py b/ipatests/test_integration/test_ipahealthcheck.py
index 7c3f5857a477070d8a9b52c04d41f35ac580c97f..05a0adb24a3f26d70d0690462e7c0fefbf98c6e6 100644
--- a/ipatests/test_integration/test_ipahealthcheck.py
+++ b/ipatests/test_integration/test_ipahealthcheck.py
@@ -1405,7 +1405,7 @@ class TestIpaHealthCheck(IntegrationTest):
         msg = "[plugin:ipa] collecting path '{}'".format(HEALTHCHECK_LOG)
         cmd = self.master.run_command(
             [
-                "sosreport",
+                "sos", "report",
                 "-o",
                 "ipa",
                 "--case-id",
@@ -1508,7 +1508,7 @@ class TestIpaHealthCheck(IntegrationTest):
         caseid = "123456"
         self.master.run_command(
             [
-                "sosreport",
+                "sos", "report",
                 "-o",
                 "ipa",
                 "--case-id",
-- 
2.51.0

